modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_pdf_generator.rb

```diff
@@ -56,7 +56,7 @@ def perform(claim_id, middle_initial, perform_async = true) # rubocop:disable Me
 
         log_job_progress(claim_id,
                          '526EZ PDF generator job finished')
-        start_docker_container_job(auto_claim&.id, perform_async) if auto_claim.status != errored_state_value
+        start_docker_container_job(auto_claim&.id) if auto_claim.status != errored_state_value
       rescue Faraday::ParsingError, Faraday::TimeoutError => e
         set_evss_response(auto_claim, e)
```

Comment:
@stiehlrod  I think you will need to include the `set_errored_state_on_claim(auto_claim)` on lines 70 & 61 in this file, otherwise I'm not sure anything will set their status to `errored`